src/viewer.js

```diff
@@ -195,6 +203,7 @@ export class Viewer {
 
     // Wait for document to become visible.
     this.docState_.onVisibilityChanged(() => {
+      this.onVisibilityResolve_();
```

Comment:
is there a better place to call the resolve? or is there an event emitted that can tell me `e.visible`?